src/app/large-display/large-display.component.html

```diff
@@ -1,52 +1,34 @@
-<ng-container [ngSwitch]="currentScreen">
```

Comment:
I'm not sure why we are removing this stuff since we will need it in the future to have different screens for different parts of a round (pre-gameplay and post-gameplay).

Reply:
The screen is redundant with the session lobby.

Reply:
Hmm... what about about something like this?

```html
  <ng-container *ngSwitchCase="ScreenId.Lobby">
    <app-session-lobby></app-session-lobby>
  </ng-container>
```

(or I suppose you could simplify that to)
```html
  <app-session-lobby *ngSwitchCase="ScreenId.Lobby"></app-session-lobby>
```

That would fix the redundancy problem w/o having to route to different pages. (The main benefit is that it's easier to manage application state if it's all on one page. Otherwise, the teacher could return to the session lobby by pressing the back button, which is a pain to have to deal with.)

Reply:
Yes but the session lobby shouldn't be its own router it should be inserted in the large display component and we will have a round lobby and such in the future as well.

Reply:
@floogulinc Could you say more about what you mean by "shouldn't be its own router"?